### PR TITLE
Issue 967 add briefcase pull start from last submission pulled info

### LINKED ISSUE #967: [Briefcase] Add resume last pull UI and CLI documentation
https://github.com/opendatakit/briefcase/pull/698 adds a new "resume last pull" feature with changes to the UI and the CLI that should be added to the docs.

### PR DESCRIPTION
**Pull request by ggalmazor**
closes #967

#### What is included in this PR?
New info about the new Briefcase feature introduced in opendatakit/briefcase#698

### CODE REVIEW

File: odk1-src/briefcase-using.rst [hunk 1]
```diff
@@ -60,6 +60,19 @@ To download blank forms and completed form instances from an :doc:`Aggregate <ag
...
     However, if your Aggregate server is :doc:`installed on Google App Engine <aggregate-app-engine>`, this setting may cause problems with large attachments. If your form has submission attachments (file uploads, videos, images, sound recordings) you should experiment with this setting and see if it improves or worsens performance.
 
+.. note::
+  :name: briefcase-start-pull-from-last-submission-pulled
+
+  .. rubric:: Start pull from last submission pulled
+
+  .. container:: details
+
+    If your forms have more than 100 submissions, you can benefit from enabling the configuration parameter :guilabel:`Start pull from last submission pulled` in the :guilabel:`Settings` tab to save some requests to your Aggregate server.
```
Comment (yanokwa) [inline odk1-src/briefcase-using.rst]: `save some requests to your Aggregate server` is precise, but not user-friendly. What about `speed up pulls` or `dramatically speed up pulls` (if true).

Comment (ggalmazor) [reply odk1-src/briefcase-using.rst]: Maybe we can steer the phrase toward scenarios with bad network connectivity or server-side costs management. My guess is that the speedup will only be noticeable on huge forms.

Comment (yanokwa) [reply odk1-src/briefcase-using.rst]: What about "to improve pull performance on slow connections"?

Comment (ggalmazor) [reply odk1-src/briefcase-using.rst]: awesome :)

**Comment by ggalmazor** [comment]
Briefcase v1.14.0 has been released today. We should merge this PR if it's OK

**Pull request merged**